### PR TITLE
Version 0.8.5.

### PR DESCRIPTION
**Pull request by xeioex**

### CODE REVIEW

**Review by thresheek — APPROVED** [review]

**Review by thresheek — APPROVED** [review]

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,53 @@
...
+       "downstream" event type for s.on() instead.
+
+    *) Feature: added timeout argument for shared dictionary methods
+       add(), set() and incr().
+
+    *) Improvement: improved checking for duplicate js_set variables.
+
+    *) Bugfix: fixed request Host header when the port is non-standard.
+
+    *) Bugfix: fixed handling of a 0 length request body in ngx.fetch()
```
Comment (y82) [inline CHANGES:20]: zero-length

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,53 @@
...
+       with "upload" or "download" event types will now convert bytes
+       invalid in UTF-8 encoding into the replacement character.  When
+       working with binary data, use r.rawVariables.var, r.requestBuffer,
+       r.responseBuffer, s.rawVariables.var, and the "upstream" or
+       "downstream" event type for s.on() instead.
+
+    *) Feature: added timeout argument for shared dictionary methods
+       add(), set() and incr().
+
+    *) Improvement: improved checking for duplicate js_set variables.
```
Comment (y82) [inline CHANGES:16]: this change also appears in 0.8.4

File: CHANGES [hunk 3]
```diff
@@ -1,3 +1,53 @@
...
+       argument.
+
+    *) Bugfix: fixed retval handling after an exception in
+       Array.prototype.toSpliced(), Array.prototype.toReversed(),
+       Array.prototype.toSorted().
+
+    *) Bugfix: fixed RegExp.prototype[@@replace]() with replacements
+       containing "$'", "$\`" and strings with Unicode characters.
+
+    *) Bugfix: fixed 1-byte overread in decodeURI() and
```
Comment (y82) [inline CHANGES:43]: a one-byte overread

File: CHANGES (hunk 2 above)
Comment (xeioex) [reply CHANGES:16]: This is a fix on top of similar change in 0.8.4.
I  will rephrase it to make it a fix.

**Review by y82 — APPROVED** [review]

**Review by thresheek — APPROVED** [review]
LGTM
